### ISSUE TITLE
'Mun Network' contract not activating

### ISSUE THREAD

**Issue by lnxbil** [jrossignol/ContractPack-RemoteTech#15]
Hi,

I build a lot of satellites and it works as for the 95% goal is reached, yet the time does not start to count down. I tried it two times (first time I canceled the contract by debug menu to get rid of it).

Text is still 'testing time after network up 7 days'. I'm running on MacOS and KSP 1.0.4. Is there a way to turn on debugging for the contracts? What can I do to assist?

Best,
Andreas

**Comment by jrossignol** [comment]
Can you confirm you're on the latest contract Configurator? There were some
bugs around this fixed not too long ago.

**Comment by lnxbil** [comment]
I think so:

```
KSP Version: 1.0.4

Installed Modules:

- Chatterer 0.9.6
- ContractConfigurator 1.7.4
- ContractConfigurator-HistoricMissions 1.5.0
- ContractConfigurator-RemoteTech 2.0.01
- CriticalTemperatureGauge 1.0.4-a
- DMagicOrbitalScience 1.0.8
- GCMonitor 1.2.6
- KerbalAlarmClock v3.4.0.0
- KerbalConstructionTime 1.2.2
- KerbalEngineerRedux 1.0.18.0
- MechJeb2 2.5.3
- ModuleManager autodetected dll
- PortraitStats 6.0
- RemoteTech v1.6.8
- SCANsat v14.2
- SigmaRTExpansion 0.4.5
- TACLS v0.11.1.20
- TACLS-Config-Stock v0.11.1.20
```

**Comment by jrossignol** [comment]
Ah, fixed in Contract Configurator 1.7.5, which isn't released yet.  Let me get one more thing done with it, and I'll release it tonight.  Keep in mind there _may_ be some backwards compatibility issues on that change (there's a possibility it won't work on the current contract even after the update, you'll just have to cancel it and move on).

**Comment by lnxbil** [comment]
Yes, it works now: clock is running

Thank you very much!
